src/ExponentialMap.jl

```diff
@@ -102,15 +102,16 @@ function get_row(spmexp::SparseMatrixExp{N}, i::Int)::RowVector{N} where {N}
     n = size(spmexp, 1)
     aux = zeros(N, n)
     aux[i] = one(N)
-    return transpose(expmv(one(N), spmexp.M.', aux))
+    Mtranspose = copy(transpose(spmexp.M))
```

Comment:
The syntax `.'` is deprecated in 0.7, and i did what they recommend, see warning messag e below: 

```julia
julia> x = fill(1.0, 3);

julia> x.'
┌ Warning: The postfix .' syntax is deprecated. For vector v in v.', use transpose(v) instead. For matrix A in A.', use copy(transpose(A)) instead, unless A.' appears as an argument of *, / or \. In those cases, use transpose(A) instead. 
└ @ nothing none:0
1×3 LinearAlgebra.Transpose{Float64,Array{Float64,1}}:
 1.0  1.0  1.0
```

Without the `copy` you modify the same object:

in 0.7:

```julia
julia> M = [1.0 -1; 1 1]
2×2 Array{Float64,2}:
 1.0  -1.0
 1.0   1.0

julia> Mt = transpose(M)
2×2 LinearAlgebra.Transpose{Float64,Array{Float64,2}}:
  1.0  1.0
 -1.0  1.0

julia> Mt[1, 1] = 5
5

julia> M
2×2 Array{Float64,2}:
 5.0  -1.0
 1.0   1.0
```

in 0.6:

```julia
julia> M = [1.0 -1; 1 1]
2×2 Array{Float64,2}:
 1.0  -1.0
 1.0   1.0

julia> Mt = transpose(M)
2×2 Array{Float64,2}:
  1.0  1.0
 -1.0  1.0

julia> Mt[1, 1] = 5
5

julia> M
2×2 Array{Float64,2}:
 1.0  -1.0
 1.0   1.0
```

Reply:
Okay, they have introduced a lazy type for transposition, I see :+1:
I guess this slows down the 0.6 version, but okay.

Reply:
But for the plot recipes we shouldn't need the copies I think. Plots should not modify their input, right?

Reply:
> Okay, they have introduced a lazy type for transposition, I see

exactly.

> I guess this slows down the 0.6 version, but okay.

the 0.6 version already does the copy internally, IIUC, so there shoulnd't be a slowdown. but it would be nice to check.

> But for the plot recipes we shouldn't need the copies I think. Plots should not modify their input, right?

i doubted about this one. ok, let's remove the `copy` here. the plot just needs to access the matrix.